sawmill-core/src/test/java/io/logz/sawmill/processors/DateProcessorTest.java

```diff
@@ -40,6 +44,27 @@ public void testUnixMsFormat() {
         assertThat(dateProcessor.process(doc).isSucceeded()).isTrue();
         assertThat((String)doc.getField(targetField)).isEqualTo(zonedDateTime.format(DateProcessor.ELASTIC));
     }
+    @Test
+    public void testUnixSFormat() {
+        String field = "datetime";
+        String targetField = "@timestamp";
+        ZoneId zoneId = ZoneId.of("Europe/Paris");
+        String messageExample = "{\"datetime\" : 1557315444.223578}";
+
+        Doc doc = new Doc(JsonUtils.fromJsonString(Map.class,messageExample));
+        Double dateTimeFromDoc = ((Double)DOUBLE.convertFrom(doc.getField("datetime")));
```

Comment:
I Do the extraction because i want to get the Storngly type field (in this case - Double) that is created from the Jackson deserialization... i'll explain better when we talk.